src/errors/dnsRecordsError.ts

```diff
@@ -0,0 +1,46 @@
+import { DnsRecordType } from '../publicTypes';
+type DnsRecordsErrorHandler = (error: DnsRecordsErrorOptions) => string;
+/** Explains DnsRecords Error options */
+type DnsRecordsErrorOptions = {
+  recordType?: DnsRecordType
+};
+
+export enum DnsRecordsErrorCode {
+  InconsistentTtl = "InconsistentTtl",
```

Comment:
According to `<subject><verb>` convention it should be `TtlInconsistent`.

Reply:
"Inconsistent" is not a verb though. For example, in HTTP error codes you'll always find adjectives before nouns

Reply:
I was thinking it should also be a follow up. At least this is how it is done in advanced libraries I used before. HTTP is full of legacy. I wouldn't use it as example.

Reply:
Alright, here's an example from GRPC:
https://github.com/grpc/grpc/blob/master/doc/statuscodes.md
It shouldn't be full of legacy. :)

The rule is to write adjectives before nouns. You might have seen participles used after subjects, like `ACCESS_DENIED`, which is correct, but this is a different case.